From 45dcb212ca992d2ae3519809f4afb153d9ea9741 Mon Sep 17 00:00:00 2001
From: Roland Kammerer <roland.kammerer@linbit.com>
Date: Fri, 12 Jul 2019 13:36:50 +0200
Subject: [PATCH] netlink: prepare for kernel v5.2

Backport the netlink from 92ade5989027ee7ea5fbde38ad67cd234822afcf

Signed-off-by: Seiya Kojima <seiya.kojima@miraclelinux.com>
---
 user/shared/libgenl.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/user/shared/libgenl.h b/user/shared/libgenl.h
index 4ade4035..b75bb5ce 100644
--- a/user/shared/libgenl.h
+++ b/user/shared/libgenl.h
@@ -23,6 +23,10 @@
 #define SOL_NETLINK 270
 #endif
 
+#ifndef NLA_F_NESTED
+#define NLA_F_NESTED 0
+#endif
+
 #define DEBUG_LEVEL 1
 
 #define dbg(lvl, fmt, arg...)				\
-- 
2.25.1

